srcpkgs/serd/template

```diff
@@ -1,14 +1,22 @@
 # Template file for 'serd'
 pkgname=serd
-version=0.30.6
+version=0.30.10
 revision=1
 build_style=waf3
 short_desc="Lightweight C library for RDF syntax"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="ISC"
 homepage="http://drobilla.net/software/serd/"
 distfiles="http://download.drobilla.net/${pkgname}-${version}.tar.bz2"
-checksum=f5a2c74c659d8b318059068f135a43a3771491c367b6947e053a713b23cd37ef
+checksum=affa80deec78921f86335e6fc3f18b80aefecf424f6a5755e9f2fa0eb0710edf
+
+if [ "${XBPS_CHECK_PKGS}" ]; then
+	configure_args+=" --test"
+fi
+
+do_check() {
+	./waf test
+}
```

Comment:
Maybe we can add this to the build style? what do you think?

Reply:
Yes I think too!

Reply:
But I don't know if I'm ready to mess around with the build styles :D

Reply:
I will leave this as a future improvement, we can check if other waf stuff also has tests.

Reply:
It looks like `waf test` is officially documented, but doesn't always exist. And I didn't find a way to check if a build script contains a test task or not, like with `make -q`